src/tapis-hooks/src/systems/useList.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+import { list } from 'tapis-api/src/systems';
+import { Systems } from '@tapis/tapis-typescript'
+import { useTapisConfig } from 'tapis-hooks/src';
+
+const useList = (params: Systems.GetSystemsRequest) => {
```

Comment:
Can we change this to `useSystemsList` so we don't have to do import aliasing if we ever have one component handling multiple listing types?

Reply:
Done